flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/job-checkpoints.component.html

```diff
@@ -551,15 +551,15 @@
             </td>
           </tr>
           <tr>
-            <td>Changelog state-backend</td>
-            <td *ngIf="checkPointConfig['state_backend'] === 'ChangelogStateBackend'">Enabled</td>
-            <td *ngIf="checkPointConfig['state_backend'] !== 'ChangelogStateBackend'">Disabled</td>
+            <td>State Changelog</td>
```

Comment:
```suggestion
            <td>Changelog State</td>
```

Reply:
I think it should actually be `State Changelog` - i.e. "Changelog of Flink State";
`Changelog State` reads as "state of changelog of something".

Reply:
Hi @Myasuka , how do you think about “Changelog of Flink State” ?